src/request.js

```diff
@@ -0,0 +1,27 @@
+export default class Request {
+  constructor(gameId) {
+    this.gameId = gameId;
+    this.base = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+  }
+
+  async addScore(name, score) {
+    const result = await fetch(`${this.base}games/${this.gameId}/scores/`, {
+      method: 'POST',
+      body: JSON.stringify({
+        user: name,
+        score,
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    })
+      .then((response) => response.json());
+    return (result.result === 'Leaderboard score created correctly.');
+  }
+
+  async getScores() {
```

Comment:
- [ ] The arrow function usage goes on here as well 🙏 
_Please see the above comment regarding this request._ 😄

Reply:
Please see my comment above.

Reply:
- [ ] Use arrow functions here as well